File: src/NocWMS-Client/src/app/models/category.ts

export class Category {
    constructor(category?: Partial<Category>) {
        if (category) {
            this.id = category.id;
            this.categoryName = category.categoryName;
            this.creator = category.creator;
            this.createTime = category.createTime;
            this.remarks = category.remarks;
        }
    }

    public id: number;
    public categoryName: string;
    public creator: string;
    public createTime: string;
    public remarks: string;
}

File: src/NocWMS-Server/Server.ApiService/Models/Category.cs
using System;
using System.Collections.Generic;
using KiraNet.Camellia.Infrastructure.DomainModel.Data;

namespace Server.ApiService.Models
{
    public class Category : IEntity<int>
    {
        public int Id { get; set; }
        public string CategoryName { get; set; }
        public string Creator { get; set; }
        public string Remarks { get; set; }
        public DateTime CreateTime { get; set; }

        public List<Inventory> Inventories { get; set; }
        public List<InboundReceiptItem> InboundReceiptItems { get; set; }
        public List<OutboundReceiptItem> OutboundReceiptItems { get; set; }
        public List<Product> Products { get; set; }
    }
}